features/dd-sdk-android-flags/src/main/kotlin/com/datadog/android/flags/FlagsClient.kt

```diff
@@ -307,14 +343,14 @@ interface FlagsClient {
         internal fun createInternal(
             configuration: FlagsConfiguration,
             featureSdkCore: FeatureSdkCore,
-            flagsFeature: FlagsFeature
+            flagsFeature: FlagsFeature,
+            name: String = DEFAULT_CLIENT_NAME
```

Comment:
is default argument needed here, given it is already there for public API?

Reply:
negative